lib/gtfs_server/agencies/trimet/controller.ex

```diff
@@ -0,0 +1,16 @@
+defmodule GtfsServer.Agencies.Trimet.Controller do
+  require Logger
+
+  import GtfsServer.Agencies.Trimet.Service, only: [dedupe_feed: 2, process_feed: 2]
+
+  alias GtfsServer.Agencies.Trimet.Client
+
+  def get_next_feed_message(feed_type) do
+    with {:ok, entities} <- Client.get_feed(feed_type) do
+      entities
+        |> dedupe_feed(feed_type)
+        |> process_feed(feed_type)
```

Comment:
need to add an intermediate step that will massage the data. I.E remove the FeedEntity wrapper and maybe add relevant transit agency id?